cachelib/navy/common/Device.cpp

```diff
@@ -923,8 +1025,16 @@ IoContext* FileDevice::getIoContext() {
     std::unique_ptr<folly::AsyncBase> asyncBase;
     if (useIoUring) {
 #ifndef CACHELIB_IOURING_DISABLE
-      asyncBase = std::make_unique<folly::IoUring>(qDepthPerContext_, pollMode,
-                                                   qDepthPerContext_);
+      if (fdpNvmeVec_.size() > 0) {
+        folly::IoUringOp::Options options;
+        options.sqe128 = true;
+        options.cqe32 = true;
```

Comment:
right, we can remove the line "options.cqe32 = true;" for now. We don't have the big_cqe use cases in the current code.

Reply:
@arungeorge83 Actually, I would like to remove this option completely. So, we don't need `sqe128`, either, right?

Reply:
@jaesoo-fb Actually both sqe128 and cqe32 are mandatory for NVMe passthrough commands. Else the kernel NVMe driver will reject those ios. 
Reference: https://elixir.bootlin.com/linux/latest/source/drivers/nvme/host/ioctl.c#L742, https://github.com/Iceber/iouring-go/issues/21

Reply:
Also from Kanchan Joshi's words (Author of the IO passthru patch - https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=456cba386e94f22fa1b1426303fdcac9e66b1417),
NVMe passthrough commands are of 72 bytes in size. This can not fit into regular SQE. Therefore big SQE (128 bytes) is required.
Also passthrough commands may return more than one result to userspace. Returning addtional result is not possible with regular CQE. Hence big CQE is needed.

Reply:
Thanks for clarification. Yeah, it looks clear here https://elixir.bootlin.com/linux/v6.7/source/drivers/nvme/host/ioctl.c#L742